datadog/fwprovider/resource_datadog_integration_gcp_sts.go

```diff
@@ -52,10 +57,16 @@ func (r *integrationGcpStsResource) Schema(_ context.Context, _ resource.SchemaR
 	response.Schema = schema.Schema{
 		Description: "Provides a Datadog Integration GCP Sts resource. This can be used to create and manage Datadog - Google Cloud Platform integration.",
 		Attributes: map[string]schema.Attribute{
+			"account_tags": schema.SetAttribute{
+				Optional:    true,
+				Description: "Tags to be associated with GCP metrics and service checks from your account.",
+				ElementType: types.StringType,
+			},
 			"automute": schema.BoolAttribute{
 				Optional:    true,
 				Computed:    true,
 				Description: "Silence monitors for expected GCE instance shutdowns.",
+				Default:     booldefault.StaticBool(false),
```

Comment:
Whats the reasoning for this change? This can potentially break existing customers if they set `automute` to true in the UI.

Reply:
Fair enough. Hadn't really considered the case where customers were using both terraform and the UI to manage their configuration. Will remove.